Platformgame/Assets/0. KimJiMin/GameManager.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.IMGUI.Controls;
+using UnityEngine;
+using UnityEngine.Serialization;
+using Random = UnityEngine.Random;
+
+// written by Kim Jimin
+public class GameManager : MonoBehaviour
+{
+    private float _distance;
+    private Vector3 _randomXVector3;
+    private Vector3 CameraPositionY; //카메라 위쪽의 Y좌표값을 가져오는 역할
+    private float prevX; //이전 x값을 저장하는 변수
+    private float intervalY;//Y 간격
```

Comment:
```suggestion
    private float _prevX; //이전 x값을 저장하는 변수
    private float _intervalY;//Y 간격
```
위 변수가 코드 컨벤션을 지키지 않고 작성 됨.